### ISSUE TITLE
input_list for snpe-dlc-quantize and wrong transcription with 0 logits for fp32 .dlc model on S24

### ISSUE THREAD

**Issue by v-prgmr** [quic/qidk#41]
Hello devs,

I am trying to run the WhisperASRHelper demo. Where do i find the list.txt which is needed for quantizing the .dlc model? 

Host OS: Linux 22.04
SNPE SDK Version: 2.27.0.240926
Deployment device: S8650 (Samsung S24 Ultra)

When I copy all the libs from SNPE SDK and Hexagon75 into the app's jnilibs folder, and use the decoder tflite model and the newly created encoder dlc model using the SNPE SDK without quantization (do not how to format the list.txt, hence tried using the f32 dlc model), the transcription is always "I I I I ....".

Appreciate your help.

Thank you

**Comment by v-prgmr** [comment]
Update: I followed the whisper_notebook.ipynb to create the input_list list.txt.

**Comment by v-prgmr** [comment]
Update:

- I followed the whisper_notebook.ipynb to create the input_list list.txt.
- Logits were 0 for fp32 model, a16w8 model on the SNPE SDK Version: 2.27.0.240926
- Logits were not 0 anymore for a16w8 model when SNPE SDK Version: 2.25.0.240728 (but still 0 for fp32 model) - all the models produce meaningful results on Linux host after conversion and quantization
